chall01/sluhtala.py

```diff
@@ -0,0 +1,30 @@
+import sys
```

Comment:
You forgot the shebang to execute the script 

https://stackoverflow.com/questions/7670303/purpose-of-usr-bin-python3/7670338